subprojects/detekt.yml

```diff
@@ -0,0 +1,689 @@
+    ModifierOrdering:
+        active: true
+        autoCorrect: true
+    MultiLineIfElse:
+        active: true
+        autoCorrect: true
+    NoBlankLineBeforeRbrace:
+        active: true
+        autoCorrect: true
+    NoConsecutiveBlankLines:
+        active: true
+        autoCorrect: true
+    NoEmptyClassBody:
+        active: true
+        autoCorrect: true
+    NoEmptyFirstLineInMethodBlock:
+        active: false
+        autoCorrect: true
+    NoLineBreakAfterElse:
+        active: true
+        autoCorrect: true
+    NoLineBreakBeforeAssignment:
+        active: true
+        autoCorrect: true
+    NoMultipleSpaces:
+        active: true
+        autoCorrect: true
+    NoSemicolons:
+        active: true
+        autoCorrect: true
+    NoTrailingSpaces:
+        active: true
+        autoCorrect: true
+    NoUnitReturn:
+        active: true
+        autoCorrect: true
+    NoUnusedImports:
+        active: true
+        autoCorrect: true
+    NoWildcardImports:
+        active: true
+    PackageName:
+        active: true
+        autoCorrect: true
+    ParameterListWrapping:
+        active: true
+        autoCorrect: true
+        indentSize: 4
+    SpacingAroundColon:
+        active: true
+        autoCorrect: true
+    SpacingAroundComma:
+        active: true
+        autoCorrect: true
+    SpacingAroundCurly:
+        active: true
+        autoCorrect: true
+    SpacingAroundDot:
+        active: true
+        autoCorrect: true
+    SpacingAroundDoubleColon:
+        active: false
+        autoCorrect: true
+    SpacingAroundKeyword:
+        active: true
+        autoCorrect: true
+    SpacingAroundOperators:
+        active: true
+        autoCorrect: true
+    SpacingAroundParens:
+        active: true
+        autoCorrect: true
+    SpacingAroundRangeOperator:
+        active: true
+        autoCorrect: true
+    SpacingBetweenDeclarationsWithAnnotations:
+        active: false
+        autoCorrect: true
+    SpacingBetweenDeclarationsWithComments:
+        active: false
+        autoCorrect: true
+    StringTemplate:
+        active: true
+        autoCorrect: true
+
+naming:
+    active: false
+    ClassNaming:
+        active: true
+        excludes: ['**/test/**', '**/androidTest/**', '**/commonTest/**', '**/jvmTest/**', '**/jsTest/**', '**/iosTest/**']
+        classPattern: '[A-Z][a-zA-Z0-9]*'
+    ConstructorParameterNaming:
+        active: true
+        excludes: ['**/test/**', '**/androidTest/**', '**/commonTest/**', '**/jvmTest/**', '**/jsTest/**', '**/iosTest/**']
+        parameterPattern: '[a-z][A-Za-z0-9]*'
+        privateParameterPattern: '[a-z][A-Za-z0-9]*'
+        excludeClassPattern: '$^'
+        ignoreOverridden: true
+    EnumNaming:
+        active: true
+        excludes: ['**/test/**', '**/androidTest/**', '**/commonTest/**', '**/jvmTest/**', '**/jsTest/**', '**/iosTest/**']
+        enumEntryPattern: '[A-Z][_a-zA-Z0-9]*'
+    ForbiddenClassName:
+        active: false
```

Comment:
As I understand If on the top level of `naming` we have active=false then we don't need to duplicate active=false

Reply:
i only changed top level one's;
this is detekt defaults